js_modules/dagster-ui/packages/ui-core/src/assets/AssetSidebarActivitySummary.tsx

```diff
@@ -100,7 +100,7 @@ export const AssetSidebarActivitySummary: React.FC<Props> = ({
             <Icon name="open_in_new" color={Colors.Link} />
           </Box>
           <Box margin={{horizontal: 24}} flex={{gap: 12, alignItems: 'flex-start'}}>
-            <Body style={{flex: 1}}>
+            <Body style={{flex: 1, marginBottom: 12}}>
```

Comment:
Adjust the `gap` on the container `Box` instead?

Reply:
Ahh good call!